hack/make-rules/test.sh

```diff
@@ -223,6 +223,10 @@ verifyAndSuggestPackagePath() {
   local original_package_path="$3"
   local suggestion_package_path="$4"
 
+  if [[ "${specified_package_path}" =~ '/...'$ ]]; then
+    specified_package_path=${specified_package_path::-4}
```

Comment:
This is local only and doesn't affect what is actually built, correct?

Reply:
@liggitt yup, that's 100% correct. It is just used for determining whether the path the user is trying to test is valid.